### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by jeremylt** [Eudicods/outworlds-wastes#113]
The responsiveness for the web view is .... not great

See the deployed website here: https://outworlds-wastes.jeremylt.org/

I like the current desktop view:
![image](https://github.com/user-attachments/assets/cf53695c-3af5-45e7-a149-26094fb273d4)

And the current mobile view is fine:
![image](https://github.com/user-attachments/assets/d6bac850-e09e-41b2-8165-b6c8db866afa)

However, it is all pretty screwy with Apple, and in any mobile view the images are way too big. Especially here:
https://outworlds-wastes.jeremylt.org/sample-forces.html#sample-forces1
![image](https://github.com/user-attachments/assets/2b1bcd66-e9b0-4675-b9aa-abeb368d6d76)

The CSS file here should be updated for better responsiveness:
https://github.com/Eudicods/outworlds-wastes/blob/main/rules/style.css

Note: This website is built using LaTex. You do not need to futz with LaTeX. The built website source is on the `rules-web` branch:
https://github.com/Eudicods/outworlds-wastes/tree/rules-web

If you have questions, you can ask here, but for a more realtime back and forth, you can find me on the freeCodeCamp Discord:
https://www.freecodecamp.org/news/freecodecamp-discord-chat-room-server/

**Comment by Eshan05** [comment]
Hello there!
I would like to propose a few changes and work on it.

Firstly, regarding the responsiveness of the web page you linked I'd make changes such that the page doesn't have any horizontally scrolling regarding the table and instead you can scroll the table itself. This makes it so that the table can be of any size but the viewing experience is consistent.

Also for images I was thinking it could be so that jt big enough to fit the width but then if you click/press on it you have an enlarged view through a modal

Tldr; 
1. No horizontal scrolling whatsoever on mobile so the viewing and navigation is better
2. Fit-to-width images and or a modal on press whichever seems better

**Comment by Jash2606** [comment]
Hey @jeremylt ,
I'd like to contribute to this issue for Hacktoberfest . Could you please assign it to me?

**Comment by uzefaqureshi06** [comment]
can you assign this to me I will do your work

**Comment by jeremylt** [comment]
@Jash2606 @uzefaqureshi06 If you have specific changes in mind, can you outline them here? I'm happy to accept multiple PRs all handling different things, as I'm sure there are multiple responsiveness issues with the site.

**Comment by uzefaqureshi06** [comment]
We can change the font family, background-color ,add border to the image  present in our application

**Comment by jeremylt** [comment]
> We can change the font family, background-color ,add border to the image present in our application

It looks like Eshan05 already has a claim on working on images

**Comment by jeremylt** [comment]
The image sizes and table scrolling are fixed. If there are other responsive issues that folks see' I'd welcome further PRs.

**Comment by jeremylt** [comment]
Oh, I just found one

https://github.com/Eudicods/outworlds-wastes/blob/rules-web/short-story.html#L233

There is a second set of images that spill to the right in mobile view, figure>p>a>img. Probably should target these in addition to figure>p>img?

**Comment by Eshan05** [comment]
> Oh, I just found one
> 
> https://github.com/Eudicods/outworlds-wastes/blob/rules-web/short-story.html#L233
> 
> There is a second set of images that spill to the right in mobile view, figure>p>a>img. Probably should target these in addition to figure>p>img?

Well didn't know about this. But you can just do `figure > p img` and it'll target any and all `img` in a `p` including those within an `a` tag

**Comment by jeremylt** [comment]
Yeah, no worries. The structure of the generated HTML can be a bit complicated sometimes. I've opened that up on a separate issue and if it's not claimed by the end of the month I'll patch it myself.

Thanks again for those fixes

**Comment by jeremylt** [comment]
Closed by #117
